src/components/Identity/createMigration.js

```diff
@@ -2,27 +2,36 @@ import { cookieJar } from "../../utils";
 import { EXPERIENCE_CLOUD_ID } from "./constants/cookieNames";
 
 export default (imsOrgId, idMigrationEnabled) => {
+  const amcvCookieName = `AMCV_${imsOrgId}`;
+
   return {
-    getEcidFromAmcvCookie(identityCookieJar) {
+    getEcidFromLegacyCookie(identityCookieJar) {
       let ecid = null;
+      const legacyItpCookieName = "s_ecid";
+
       if (idMigrationEnabled) {
-        const amcvCookieValue = cookieJar.get(`AMCV_${imsOrgId}`);
-        if (amcvCookieValue) {
+        const legacyEcidCookieValue =
```

Comment:
I just want to point out that if the legacy itp cookie doesn't match the format, we don't even look at the amcv cookie.  I don't know enough about these cookies to know if this is an issue or not.

Reply:
Thank you Jon. The legacy ITP can only have the correct format because the ECID is the only value that goes in there. That being said, I did consider going over the whole workflow one cookie at a time, so I might do that anyway, but in a separate PR.